src/admin/dto/update-admin.dto.ts

```diff
@@ -1,4 +1,11 @@
 import { PartialType } from "@nestjs/mapped-types";
+import { Exclude } from "class-transformer";
 import { CreateAdminDto } from "./create-admin.dto";
 
-export class UpdateAdminDto extends PartialType(CreateAdminDto) {}
+export class UpdateAdminDto extends PartialType(CreateAdminDto) {
```

Comment:
in the front-end, set email input disabled, and this is the back-end way to prevent a user to modify email and permission? is it right?

Reply:
Yes. in my impression, our tutor seems to have said that the email cannot be changed.